config/settings.yml

```diff
@@ -862,6 +862,7 @@
   :level_policy: info
   :level_remote_console: info
   :secret_filter: []
+:messaging_type: kafka
```

Comment:
@agrare @nasark Will we likely have other options related to messaging after this?  Wondering if it's better to have a simple nested structure now... Something like:

```
:messaging:
  :type: kafka
```